arrows/core/detected_object_set_input_simulator.cxx

```diff
@@ -1,5 +1,5 @@
 /*ckwg +29
- * Copyright 2018-2020 by Kitware, Inc.
+ * Copyright 2018, 2020 by Kitware, Inc.
```

Comment:
Should probably still be 2018-2020
```suggestion
 * Copyright 2018-2020 by Kitware, Inc.
```

Reply:
IIRC this was a straight revert. Also, this file doesn't seem to have been edited in 2019, so the above is correct. (But see previous comment on copyrights.)